playbooks/aws/openshift-cluster/tasks/launch_instances.yml

```diff
@@ -99,6 +127,8 @@
       env: "{{ env }}"
       host-type: "{{ host_type }}"
       env-host-type: "{{ env_host_type }}"
+      node-type: "{{ node_type }}"
```

Comment:
I'm torn, leaning towards sub-host-type, but I'd probably be fine with sub-type too. Committers choice. :)